src/NServiceBus.Core/OpenTelemetry/Metrics/MessagingMetricsMeters.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace NServiceBus;
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using Pipeline;
+
+class MessagingMetricsMeters
+{
+    public MessagingMetricsMeters(IMeterFactory meterFactory)
```

Comment:
If we keep this approach can we rename this to MessagingMetrics? Or MessagingMeters? I don't have a strong preference as long as it doesn't combine both namings